tests/tests/market_tests.cpp

```diff
@@ -478,17 +484,18 @@ BOOST_AUTO_TEST_CASE(hardfork_core_453_test)
    // generate a block
    generate_block();
 
-
 } FC_LOG_AND_RETHROW() }
 
 /***
  * Tests (big) limit order matching logic after #625 got fixed
  */
 BOOST_AUTO_TEST_CASE(hardfork_core_625_big_limit_order_test)
 { try {
-   auto mi = db.get_global_properties().parameters.maintenance_interval;
-   generate_blocks(HARDFORK_CORE_625_TIME - mi); // assume all hard forks occur at same time
-   generate_blocks(db.get_dynamic_global_properties().next_maintenance_time);
+
+   if(hf1270)
+      generate_blocks(HARDFORK_CORE_1270_TIME);
+   else
+      generate_blocks(HARDFORK_CORE_625_TIME);
```

Comment:
To be more accurate, here we should generate blocks until the maintenance time but not the hf time.